GradedModulePresentationsForCAP/gap/GradedModulePresentations.gi

```diff
@@ -1099,6 +1105,78 @@ InstallGlobalFunction( ADD_GRADED_INTERNAL_HOM_ON_MORPHISMS_RIGHT,
         
     end );
 
+end );
+
+##
+InstallGlobalFunction( ADD_GRADED_UNITOR,
+                      
+  function( category )
+    local unitor_func;
+    
+    unitor_func := function( A, B )
+        return IdentityMorphism( A );
+    end;
+    
+    AddLeftUnitorWithGivenTensorProduct( category, unitor_func );
+    
+    AddRightUnitorWithGivenTensorProduct( category, unitor_func );
+    
+end );
+
+##
+InstallGlobalFunction( ADD_GRADED_ASSOCIATOR_LEFT,
```

Comment:
If it makes sense, you could fallback here to the non-graded associator. Although it probably does not simplify the code, it might encourage a more strict nesting/towering of the categories.

Reply:
I actually would not like to encourage building graded modules on top of non-graded modules. This strategy has actually led to some trouble in the past, see, e.g., the discussion about colifts in #141